spec/components/guardian_spec.rb

```diff
@@ -610,7 +625,8 @@
     end
 
     describe "private messages" do
-      fab!(:user) { Fabricate(:user, trust_level: TrustLevel[2]) }
+      SiteSetting.min_trust_level_to_allow_invite = 2
```

Comment:
I am surprised this works, as this code is not evaluated during the test run. This statement should be moved inside the `before` below.

Reply:
@eviltrout said changes are done